tools/torch_quant/torch_quant/observer.py

```diff
@@ -73,7 +74,7 @@ def __init__(self, dtype: torch.dtype = torch.qint8,
         self.register_buffer("min_val", torch.tensor(float("inf")))
         self.register_buffer("max_val", torch.tensor(float("-inf")))
         self.register_buffer("scale", torch.tensor(1.))
-        self.register_buffer("zero_point", torch.tensor(0.))
+        self.register_buffer("zero_point", torch.tensor(0, dtype=torch.int32))
```

Comment:
Should we support lower versions of torch. In earlier versions of torch, the zp is of `torch.long` dtype

Reply:
I can't make a decision right now. maybe we can discuss how to support old versions (and which versions) when some use cases come out 😂